### PR TITLE
merge unit and regression coverage

### PR DESCRIPTION
**Pull request by stevesims**
adds a `coverage` command to package.json (and several support commands) that will generate a merged coverage report from both unit tests and visual regression tests (storyshots)

**Checklist**:

* [ ] Documentation N/A
* [ ] Tests N/A
* [x] Ready to be merged

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -123,6 +128,21 @@
...
+    "branches": 95,
+    "functions": 90,
+    "lines": 95,
+    "statements": 95,
+    "watermarks": {
+      "branches": [90, 95],
+      "functions": [90, 90],
+      "lines": [90, 95],
+      "statements": [85, 95]
+    }
```
Comment (stevesims) [inline package.json:144]: these values are all kinda arbitrary for now

the purpose of this is to ensure that we get red (and yellow) colouring in the report for individual files

**Comment by stevesims** [comment]
hmm - dependabot automatically updated @storybook/react which has screwed up our storybook - there's no stories being shown any more 😞

**Comment by stevesims** [comment]
updating all the storybook modules seems to fix storybook (seemed not to work for me for a while - there were a few hiding that I didn't spot - doh!)

File: packages/storybook/package.json [hunk 2]
```diff
@@ -49,11 +49,11 @@
...
-    "@storybook/addon-docs": "6.2.9",
-    "@storybook/addon-knobs": "6.2.9",
-    "@storybook/addon-links": "6.3.2",
-    "@storybook/addons": "6.2.9",
+    "@storybook/addon-actions": "6.3.5",
+    "@storybook/addon-docs": "6.3.5",
+    "@storybook/addon-knobs": "6.3.0",
+    "@storybook/addon-links": "6.3.5",
+    "@storybook/addons": "6.3.5",
     "@storybook/react": "6.3.5",
```
Comment (penx) [inline packages/storybook/package.json:57]: hmm we're on 6.3.5 already are we? 🤔 

I take it you've searched for "6.2.9" and updated everywhere?

Why is addon knobs only updated to 6.3.0?

**Comment by penx** [comment]
> hmm - dependabot automatically updated @storybook/react which has screwed up our storybook - there's no stories being shown any more 😞

Is there something our CI should have done to catch that?

**Comment by penx** [comment]
approved other than

- question on knobs being 6.3.0
- any quick win on getting CI to have prevented the storybook auto merge? Or create a ticket?

File: packages/storybook/package.json (hunk 2 above)
Comment (stevesims) [reply packages/storybook/package.json:57]: I tried to update addon-knobs too, but found it's been deprecated and 6.3.0 is the latest version - replacement is addon-controls

**Comment by stevesims** [comment]
yeah I think we should probably create a ticket to think a bit more about potentially preventing future issues with rogue storybook (or other) module updates causing problems.

for the storybook update, it passed as chromatic didn't seem to get CI checked for that PR - https://github.com/govuk-react/govuk-react/runs/3141899601 - which makes some sense as otherwise dependabot would thrash our chromatic usage, but conversely if we did check chromatic for dependabot storybook module updates (only) that would have caught this

File: packages/storybook/package.json (hunk 2 above)
Comment (penx) [reply packages/storybook/package.json:57]: oh right

**Comment by penx** [comment]
> yeah I think we should probably create a ticket to think a bit more about potentially preventing future issues with rogue storybook (or other) module updates causing problems.
> 
> for the storybook update, it passed as chromatic didn't seem to get CI checked for that PR - https://github.com/govuk-react/govuk-react/runs/3141899601 - which makes some sense as otherwise dependabot would thrash our chromatic usage, but conversely if we did check chromatic for dependabot storybook module updates (only) that would have caught this

#907

**Review by penx — APPROVED** [review]

**Pull request merged**